introduction.md

```diff
@@ -0,0 +1,12 @@
+This file holds the text to accompany the [introduction](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504/9781492041504-video321606) video.
+
+# Introduction
+
+This Learning Path gives an introduction to inner source.
```

Comment:
`gives` seems like the wrong verb here.  instructs, teaches, provides perhaps?

Reply:
If this is below the video itself then it might makes sense to just come out and say `The video introduces inner source` instead of the this learning path.

Reply:
I'll check to see what is the placement of text in relation to the videos in the learning path.

Reply:
It looks like the text stands alone and apart from the video.  Here is [an example](https://www.safaribooksonline.com/learning-paths/learning-path-lean/9781491999738/9781491946527-/part01ch01.html) of a sample text segment in an O'Reilly learning path.